packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -823,7 +841,7 @@ contract Colosseum is Initializable, ISemver {
         bytes32 _l2ToL1MessagePasserCodeHash,
         bytes32 _messagePasserStorageRoot,
         bytes32 _stateRoot
-    ) private view {
+    ) internal view {
```

Comment:
Why did you change all private functions to internal?

Reply:
I changed all private "view" functions to internal functions. Initially the purpose was to utilize those functions in test contract(see `Colosseum.t.sol`), otherwise duplicate codes should be added to the test contract. Also I thought if we create another contract which inherits this contract in the future, it might be useful to have those view functions internally. It does not increase any code size, and just view functions will not have any side effects in my opinion.

Reply:
good point